### PR TITLE
Update version to 1.3.0 for release

### PR DESCRIPTION
**Pull request by antoninbas**

### CODE REVIEW

File: bazel/example/WORKSPACE.bazel [hunk 1]
```diff
@@ -11,8 +11,8 @@ local_repository(
 
 # http_archive(
 #     name = "com_github_p4lang_p4runtime",
-#     urls = ["https://github.com/p4lang/p4runtime/archive/v1.2.0.tar.gz"],
-#     strip_prefix = "p4runtime-1.2.0/proto",
+#     urls = ["https://github.com/p4lang/p4runtime/archive/v1.3.0.tar.gz"],
+#     strip_prefix = "p4runtime-1.3.0/proto",
```
Comment (smolkaj) [inline bazel/example/WORKSPACE.bazel:15]: Also update `tag = "v1.2"` to `1.3` below.

**Review by smolkaj — APPROVED** [review]
Thanks, that was fast!

File: bazel/example/WORKSPACE.bazel (hunk 1 above)
Comment (antoninbas) [reply bazel/example/WORKSPACE.bazel:15]: shouldn't the tag be v1.2.0 / v1.3.0? There is no v1.2 github tag right now

Comment (smolkaj) [reply bazel/example/WORKSPACE.bazel:15]: Good point!

Comment (antoninbas) [reply bazel/example/WORKSPACE.bazel:15]: Fixed

**Pull request merged**